File: smalivm/src/main/java/org/cf/smalivm/opcode/ReturnVoidOpFactory.java
package org.cf.smalivm.opcode;

import gnu.trove.map.TIntObjectMap;

import org.cf.smalivm.VirtualMachine;
import org.jf.dexlib2.builder.MethodLocation;

public class ReturnVoidOpFactory implements OpFactory {

    @Override
    public Op create(MethodLocation location, TIntObjectMap<MethodLocation> addressToLocation, VirtualMachine vm) {
        return new ReturnVoidOp(location);
    }

}

File: smalivm/src/main/java/org/cf/smalivm/opcode/ReturnVoidOpFactory.kt
package org.cf.smalivm.opcode

import org.cf.smalivm.VirtualMachine
import org.jf.dexlib2.builder.MethodLocation

class ReturnVoidOpFactory : OpFactory {
    override fun create(location: MethodLocation, addressToLocation: Map<Int, MethodLocation>, vm: VirtualMachine): Op {
        return ReturnVoidOp(location)
    }
}
